File: PostManagement/Models/Post.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace PostManagement.Models
{
    public class Post
    {
        public int PostId { get; set; }
        public AppUser appUser { get; set; }
        [Column("AuthorId")]
        public int AppUserId { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public int PublishStatus { get; set; }
        public PostCategory postCategory { get; set; }
        [Column("CategoryId")]
        public int PostCategoryId { get; set; }
    }
}

File: PostManagement/wwwroot/js/Post.js
$(() => {
    LoadProdData();
    var connection = new signalR.HubConnectionBuilder().withUrl("/signalrServer").build();
    connection.start();

    connection.on("LoadProducts", function () {
        LoadProdData();
    })
    LoadProdData();

    function LoadProdData() {
        var tr = '';
        $.ajax({
            url: '/Home/GetProducts',
            method: 'GET',
            success: (result) => {
                $.each(result, (k, v) => {
                    tr +=
                        `<tr> 
                        <td> ${v.appUser.fullName}</td>
                        <td> ${v.title}</td>
                        <td> ${v.content.trim()}</td>
                        <td> ${v.createdDate}</td>
                        <td> ${v.updatedDate}</td>
                        <td> ${v.publishStatus == 1 ? `ok` : `Not ok`}</td>
                        <td> ${v.postCategory.categoryName}</td>
                        <td>
                            <a href='/Home/detail?id=${v.postId}'>Details</a> |
                            <a href='/Home/update?id=${v.postId}'>Edit</a> |
                            <a href='/Home/delete?id=${v.postId}'>Delete</a> 
                        </td>
                    </tr>`
                })
                $("#tablePost").html(tr);
            },
            error: (error) => {
                console.log(error)
            }
        });
    }
})